src/app/pages/index.js

```diff
@@ -10,6 +10,8 @@ export const MediaAssetPage = loadable(() => import('./MediaAssetPage'));
 
 export const MostReadPage = loadable(() => import('./MostReadPage'));
 
+export const MostWatchedPage = loadable(() => import('./MostWatchedPage'));
```

Comment:
I believe there is also a failing bundle test, now that we have added the MostWatchedPage